docs/source/tutorials/hemeraIn5min.rst

```diff
@@ -146,12 +146,13 @@ If this fails, read the manual or ask a colleague.
 
 After a successfull build, run (still on the compute node, still inside your scenario directory)::
 
-  tbg -s bash -c etc/picongpu/1.cfg -t etc/picongpu/bash/mpiexec.tpl $SCRATCH/tinkering/try01/run01
+  tbg -s bash -t $PICSRC/etc/picongpu/bash/mpiexec.tpl -c /etc/picongpu/1.cfg $SCRATCH/tinkering/try01/run01
```

Comment:
@PrometheusPi The initial `/` of the `-c` option is not supposed to be there!

Reply:
@steindev please provide directly a fix that this not get lost